### PR TITLE
Remove pod stuff. Fix AirMaps.xcodeproj related to missing pod stuff

### PR DESCRIPTION
**Pull request by gilbox**
Fix #620 

My GoogleMaps PR added some unnecessary Podfile stuff to the `ios/` dir. This removes it to avoid errors related to that stuff.

@lelandrichardson @spikebrehm @felipecsl

### CODE REVIEW

File: ios/AirMaps.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -22,14 +22,8 @@
 		1125B2E61C4AD3DA007D0023 /* AIRMapPolylineManager.m in Sources */ = {isa = PBXBuildFile; fileRef = 1125B2D61C4AD3DA007D0023 /* AIRMapPolylineManager.m */; };
 		1125B2E71C4AD3DA007D0023 /* RCTConvert+MoreMapKit.m in Sources */ = {isa = PBXBuildFile; fileRef = 1125B2D91C4AD3DA007D0023 /* RCTConvert+MoreMapKit.m */; };
 		1125B2F21C4AD445007D0023 /* SMCalloutView.m in Sources */ = {isa = PBXBuildFile; fileRef = 1125B2F11C4AD445007D0023 /* SMCalloutView.m */; };
-		DA6C26381C9E2AFE0035349F /* AIRMapUrlTile.m in Sources */ = {isa = PBXBuildFile; fileRef = DA6C26371C9E2AFE0035349F /* AIRMapUrlTile.m */; settings = {ASSET_TAGS = (); }; };
-		DA6C263E1C9E324A0035349F /* AIRMapUrlTileManager.m in Sources */ = {isa = PBXBuildFile; fileRef = DA6C263D1C9E324A0035349F /* AIRMapUrlTileManager.m */; settings = {ASSET_TAGS = (); }; };
```
Comment (spikebrehm) [inline ios/AirMaps.xcodeproj/project.pbxproj]: These `AIRMapUrlTile` lines should stay.

Comment (gilbox) [inline ios/AirMaps.xcodeproj/project.pbxproj]: oh i just realized they were moved below

**Comment by gilbox** [comment]
@spikebrehm i think this is ready

**Comment by spikebrehm** [comment]
@gilbox where did you land in terms of creating a separate xcodeproj with the Google maps classes?

**Comment by gilbox** [comment]
@spikebrehm more complicated than we thought

**Pull request merged**